src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -68,14 +70,20 @@ function formatDigit(num) {
 }
 
 /**
- * Format a date object into DD/MM/YYYY format
+ * Format a date object into D, dd M, yy format
  * @param date
  * @returns {String}
  */
+
+function convertDateToDddMMMYYYY(date) {
+    return `${days[date.getDay()]}, ${formatDigit(date.getDate())} ${monthNameShort[date.getMonth()]},
+            ${date.getFullYear()}`;
```

Comment:
Use `moment.js` for this instead.

Reply:
@whipermr5 Can you guide me how to import moment.js , I tried `<script src="https://cdn.jsdelivr.net/npm/moment@2.19.1/min/moment.min.js"></script>` in `instructorFeedback.js` and it is not working.

Reply:
@whipermr5 I added line `/* global moment:false */`to the file, and it worked but I don't understand how.

Reply:
@nidhi98gupta 
https://eslint.org/docs/user-guide/configuring#specifying-globals

http://momentjs.com/docs/#/use-it/node-js/

Reply:
@LiHaoTan I think there must be a reason why we're using globals in the existing code though:
https://github.com/TEAMMATES/teammates/blob/19a0e8ee0e7d6a9e609de972257060fdee2d15f9/src/main/webapp/dev/js/common/timezone.js#L1

Reply:
Yeah I didn't check why we are doing that